packages/angular/cli/src/utilities/completion.ts

```diff
@@ -72,12 +73,29 @@ Ok, you won't be prompted again. Should you change your mind, the following comm
   // Notify the user autocompletion was set up successfully.
   logger.info(
     `
-Appended \`source <(ng completion script)\` to \`${rcFile}\`. Restart your terminal or run the following to autocomplete \`ng\` commands:
+${tags.oneLine`
+  Appended \`source <(ng completion script)\` to \`${rcFile}\`. Restart your terminal or run the
+  following to autocomplete \`ng\` commands:
+`}
 
     ${colors.yellow(`source <(ng completion script)`)}
     `.trim(),
   );
 
+  if ((await hasGlobalCliInstall()) === false) {
+    logger.warn(
+      `
+${tags.oneLine`
+  Setup completed successfully, but there does not seem to be a global install of the Angular CLI.
+  For autocompletion to work, the CLI will need to be on your \`$PATH\`, which is typically done
+  with the \`-g\` flag in \`npm install -g @angular/cli\`.
+`}
+
+See https://angular.io/cli/completion#global-install for more info.
```

Comment:
I believe that information CTAs are all in the below format.

```suggestion
For more information, see https://angular.io/cli/completion#global-install
```